### PR TITLE
CONSOLE-3909: i18n upload/download routine task - chore(i18n): update translations

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
Adding latest translations from Memsource project https://cloud.memsource.com/web/project2/show/pYEK5O39xbEhfF2ZSh1zn1

### CODE REVIEW

File: frontend/packages/topology/locales/zh/topology.json [hunk 1]
```diff
@@ -151,7 +151,7 @@
   "Find by label...": "按标签查找......",
   "Find by name...": "按名称查找......",
   "Find by name": "按名称查找",
-  "Search results may appear outside of the visible area. <2>Click here</2> to fit to the screen.": "搜索结果可能会出现在可见区域之外，<2>点这里</2>来匹配屏幕。",
+  "Search results may appear outside of the visible area. <2>Click here</2> to fit to the screen.": "搜索结果可能会出现在可见区域之外，</2>点这里 <2>来匹配屏幕。",
```
Comment (sg00dwin) [inline frontend/packages/topology/locales/zh/topology.json:154]: The numbering here looks incorrect 
```suggestion
  "Search results may appear outside of the visible area. <2>Click here</2> to fit to the screen.": "搜索结果可能会出现在可见区域之外，<2>点这里 </2>来匹配屏幕。",
```

**Review by TheRealJon — COMMENTED** [review]
Is it expected that Chinese translations for the following keys are reverted to English?
- "Disabled"
- "Size"
- "Serial"
- "Model"
- "Available"
- "Speed"
- "Registering"
- "Provisioning"
- "Provisioned"
- "Inspecting"
- "Deprovisioning"

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve
/retest

**Comment by cyril-ui-developer** [comment]
> Is it expected that Chinese translations for the following keys are reverted to English?
> 
> * "Disabled"
> * "Size"
> * "Serial"
> * "Model"
> * "Available"
> * "Speed"
> * "Registering"
> * "Provisioning"
> * "Provisioned"
> * "Inspecting"
> * "Deprovisioning"

@TheRealJon Good catch. No. It was an error from the translations backend.
 I have updated the PR. PTAL.

**Review by jhadvig — APPROVED** [review]
/lgtm

**Pull request merged**